docs/user/commands.adoc

```diff
@@ -8,6 +8,10 @@
 
 Show the cache folder of Spyglass in the platform-specific explorer program.
 
+=== `spyglassmc.showOutput`
```

Comment:
this file not being alphabetical is killing me 😭

Reply:
https://github.com/SpyglassMC/Spyglass/issues/1553